rakelib/test.rake

```diff
@@ -58,6 +57,10 @@ namespace "test" do
   desc "run all installed plugins specs"
   task "plugins"  => "bootstrap" do
     plugins_to_exclude = ENV.fetch("EXCLUDE_PLUGIN", "").split(",")
+    # the module LogStash::PluginManager requires the file `lib/pluginmanager/plugin_aliases.yml` is created
```

Comment:
the module LogStash::PluginManager requires the file `lib/pluginmanager/plugin_aliases.yml` ~~is created~~